### PR TITLE
Converted all to Python & Robocorp library

### PR DESCRIPTION
**Pull request by cosmogp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,202 +1,93 @@
...
+> The selector value is usually pretty difficult to construct. As such, you can use the `Robocorp Web Inspector`
+> from the [Robocorp Code VSCode Extension](https://marketplace.visualstudio.com/items?itemName=robocorp.robocorp-code)
+> to build valid selectors with ease.
 
 ## Summary
 
-You executed a web scraper robot, congratulations!
+You executed a web scraper task, congratulations!
 
 During the process, you learned some concepts and features of the Robot Framework and some good practices:
```
Comment (kariharju) [inline README.md]: Robot Framework mentioned

File: robot.yaml [hunk 2]
```diff
@@ -1,22 +1,10 @@
...
-      - tasks.robot
-  Web scraper Python:
-    command:
-      - python
-      - tasks.py
+  # Task names here are used when executing the bots, so renaming these is recommended.
+  Web Scrapper Top 10 Crypto:
+    shell: python -m robocorp.tasks run tasks.py
 
 condaConfigFile: conda.yaml
```
Comment (kariharju) [inline robot.yaml]: Replace `condaConfigFile: conda.yaml` with:
```
environmentConfigs:
  - environment_windows_amd64_freeze.yaml
  - environment_linux_amd64_freeze.yaml
  - environment_darwin_amd64_freeze.yaml
  - conda.yaml
```

File: tasks.py [hunk 3]
```diff
@@ -1,22 +1,86 @@
...
-if __name__ == "__main__":
-    main()
+            # save the content to the csv
+            csv_content.append(f'{index},{crypto_ticker},"{crypto_value}"')
+
+            # calculate how to space things to have a table look
+            i_spaces = 3 - len(str(index))
+            tab_spaces = 25 - len(crypto_ticker)
+            print(
+                f"### {index}{' '*i_spaces}| {crypto_ticker}{' '*tab_spaces}| $ {crypto_value}"
```
Comment (kariharju) [inline tasks.py:73]: Cosmetic:
Printing the lines as-we-go looks good on the console, but in the log.html gets a bit funky to decipher.

Maybe collect the results strings and at the end post one log.info or print:
```
            result_lines += f"### {index}{' '*i_spaces}| {crypto_ticker}{' '*tab_spaces}| $ {crypto_value} \n"

        result = "#" * 50 + '\n'
        result += "### Top 10 Cryptocurrencies:\n"
        result += "#" * 50 + '\n'
        result += result_lines
        result += "#" * 50
        print(result)
```

Comment (cosmogp) [reply tasks.py:73]: I honestly think it's better this way. 
The log.html looks quite good and you can see exactly how the loop works and how it outputs things. 
If you have a hard opinion on this, I'll change, let me know.

**Review by kariharju — APPROVED** [review]

**Pull request merged**